openstates/ar/legislators.py

```diff
@@ -47,9 +47,6 @@ def scrape_member(self, chamber, term, member_url):
         full_name = ' '.join(name_and_party[1:-1])
 
         party = name_and_party[-1]
-        if party == "()":
-            self.warning("Uch, bad party. Ducking")
-            return
 
         if party == '(R)':
```

Comment:
A complicated if/then/else can be replaced by a dict with a mapping of before:after.